examples/core/fragment/package.json

```diff
@@ -7,7 +7,11 @@
     "luma.gl": "^5.0.0"
   },
   "devDependencies": {
-    "webpack-dev-server": "^2.2.0",
-    "webpack": "^2.2.0"
+    "babel-core": "^6.26.0",
```

Comment:
Our examples are designed to run without transpilation on modern browsers. We don't want to clutter all of them with transpilation tools. Especially a multidependency setup like babel (buble is a little better). But we can certainly have one example specifically for IE11.

Reply:
ok- I'll revert this for now since it's not clear in this case why we're transpiling this one example and not others. I can add a different "IE" example that's clearly labeled if you think it would be helpful to the project.